src/devices/bus/hp_dio/hp98603.h

```diff
@@ -0,0 +1,40 @@
+// license:BSD-3-Clause
+// copyright-holders:Sven Schnelle
+
+#ifndef MAME_BUS_HPDIO_98603_H
+#define MAME_BUS_HPDIO_98603_H
+
+#pragma once
+
+#include "hp_dio.h"
+#include "bus/generic/slot.h"
+
+class dio16_98603_device :
+		public device_t,
+		public device_dio16_card_interface
+{
+public:
+	// construction/destruction
+	dio16_98603_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
+	required_device<generic_slot_device> m_romslot;
```

Comment:
Is there any reason that this data member needs to be public?  Generally data members should be protected or private if possible.